notebooks/notebooks/paper/defense-win-rate-heatmap.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.            fig, axs = plt.subplots(
</pre><p>all of this is just an indent change</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>